pkgs/test_core/lib/src/runner.dart

```diff
@@ -279,6 +280,14 @@ class Runner {
       }
     })).map((loadSuite) {
       return loadSuite.changeSuite((suite) {
+        // TODO: Support this for browser tests
+        if (suite.platform.runtime.isBrowser &&
```

Comment:
I think we don't support it for node tests either right?

It's probably more accurate to test `isJS` than `isBrowser`.

Reply:
Good point